apps/sptribs/sptribs-cron-migrate-case-links/aat/00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sptribs-cron-migrate-case-links
+spec:
+  releaseName: sptribs-cron-migrate-case-links
+  values:
+    job:
+      schedule: "20,50 * * * *"
```

Comment:
Aren't you going to have this one in the afternoon?
20,50 12-16 * * *

Reply:
Updated to 16-18 for Case links as the case api task code is still in progress for Links.